### DISCUSSION TITLE
Are you going to keep developing it?

### DISCUSSION THREAD

**Discussion by TheNomad11** [cu/silicon#6] [General]
I am one of many users over at Reddit who got excitied about your project. I have started playing a bit with it and I really like it. So minimalistic and clean and so comfortable for writing! But there are many projects popping up there on /r/selfhosted and many of them get abandoned sooner or later. So I just wonder about the current status. I remember vaguely you created it to use it for the rest of your life?

**Comment by cu** [comment]
Hi!

You remember correct! I use Silicon every single day, both for work and for my personal notes. The reason there isn't much activity in the repo is because I don't have much that I want to add or change right now. It's just working. I suspect a couple of dependencies could be updated, and there are some TODO items in the README for when I (or someone else) gets bored, but nothing terribly urgent.

**Reply by TheNomad11** [reply]
Great, good to hear, thanks for answering!
